### DISCUSSION TITLE
Help with making a minecraft 3d skin

### DISCUSSION THREAD

**Discussion by Kowersky** [zauberzeug/nicegui#3748] [Q&A]
### Question

Hi. I'm making a minecraft related app and I got the idea to make a 3d version of the skin in the gui but I have no clue how it could be made from a picture like the one below. I mean something like it is on namemc.
![38fa699c208462f1](https://github.com/user-attachments/assets/8916ea37-b359-4dc0-af91-5fb4e017bfec)
![image](https://github.com/user-attachments/assets/f23cde25-f75b-4235-8e0e-3db4b35f5fcc)

**Comment by rodja** [comment]
It should be possible with `scene.texture` which takes the URL of the texture image and the texture coordinates as parameter. Also we allow to load glTF files with `scene.gltf`.
